contribs/gmf/src/services/permalink.js

```diff
@@ -645,6 +651,12 @@ gmf.Permalink.prototype.handleLayersRemove_ = function(evt) {
  */
 gmf.Permalink.prototype.registerLayer_ = function(layer, opt_init) {
 
+  if (layer instanceof ol.layer.Group) {
+    layer.getLayers().forEach(function(layer) {
+      this.registerLayer_(layer, opt_init);
+    }, this)
+  }
+
```

Comment:
When a layer is `ol.layer.Group`, the code carries on after looping inside its layers, which means that events are registered for the `ol.layer.Group`.  Is this needed ?